src/hal/src/adapter.rs

```diff
@@ -51,11 +51,13 @@ pub struct MemoryProperties {
 
 /// Represents a physical device (such as a GPU) capable of supporting the given backend.
 pub trait PhysicalDevice<B: Backend>: Any + Send + Sync {
-    /// Create a new logical device.
+    /// Create a new logical device with the requested features. If `requested_features` is
+    /// `None` than only the core features are supported.
```

Comment:
wouldn't `None` be equivalent of passing the empty bit mask? in this case, let's not have `Option` at all

Reply:
I've changed it to take a Features bitflag directly now.